### PR TITLE
Consolidated unit test files; renamed test methods to conform to nami…

### PR DESCRIPTION
**Pull request by feroxi-s**
…ng conventions; added trailing commas

### CODE REVIEW

File: app/src/main/java/com/ucmobiledevelopment/freeways/dto/Incident.kt [hunk 1]
```diff
@@ -12,5 +12,5 @@ data class Incident(
     var longitude: String = "",
     var way1: String = "",
     var way2: String = "",
-    var vehiclesInvolved: Int = 0
+    var vehiclesInvolved: Int = 0,
```
Comment (feroxi-s) [inline app/src/main/java/com/ucmobiledevelopment/freeways/dto/Incident.kt:15]: Trailing commas, while not required, are an easy way to reduce future technical debt: https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

File: app/src/main/java/com/ucmobiledevelopment/freeways/dto/IncidentDTO.kt [hunk 2]
```diff
@@ -15,5 +15,5 @@ data class IncidentDTO(
     val TOTALVEHICLES: String? = "",
     val TWAY_ID: String? = "",
     val TWAY_ID2: String? = "",
-    val VE_FORMS: String? = ""
+    val VE_FORMS: String? = "",
```
Comment (feroxi-s) [inline app/src/main/java/com/ucmobiledevelopment/freeways/dto/IncidentDTO.kt:18]: Trailing commas, while not required, are an easy way to reduce future technical debt: https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

File: app/src/main/java/com/ucmobiledevelopment/freeways/dto/IncidentListDTO.kt [hunk 3]
```diff
@@ -4,5 +4,5 @@ data class IncidentListDTO(
     val Count: Int = 0,
     val Message: String = "",
     val Results: List<List<IncidentDTO>>,
-    val SearchCriteria: String = ""
+    val SearchCriteria: String = "",
```
Comment (feroxi-s) [inline app/src/main/java/com/ucmobiledevelopment/freeways/dto/IncidentListDTO.kt:7]: Trailing commas, while not required, are an easy way to reduce future technical debt: https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

File: app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt [hunk 4]
```diff
@@ -94,4 +99,30 @@ class IncidentUnitTest {
         assertTrue(containsIncidentWithCaseId400132)
     }
 
+    @Test
```
Comment (feroxi-s) [inline app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt:102]: Consolidated unit tests into one file; easier to keep track of, reducing technical debt

File: app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt [hunk 5]
```diff
@@ -94,4 +99,30 @@ class IncidentUnitTest {
...
+    fun `Given incident API is available When I fetch incidents from 2019 to 2020 in state 40 county 1 Then results should contain a case with caseId 400434`() = runTest{
+        givenIncidentServiceIsInitialized()
+        whenIncidentDataAreReadAndParsed()
+        thenTheIncidentCollectionShouldContainIncidentWithCaseId400434()
+    }
+
+    private fun givenIncidentServiceIsInitialized() {
```
Comment (feroxi-s) [inline app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt:109]: Function names now in lowerCamelCase: https://kotlinlang.org/docs/coding-conventions.html#function-names

File: app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt [hunk 6]
```diff
@@ -94,4 +99,30 @@ class IncidentUnitTest {
...
+        incidentService = IncidentService()
+    }
+
+    private suspend fun whenIncidentDataAreReadAndParsed() {
```
Comment (feroxi-s) [inline app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt:113]: Function names now in lowerCamelCase: https://kotlinlang.org/docs/coding-conventions.html#function-names

File: app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt [hunk 7]
```diff
@@ -94,4 +99,30 @@ class IncidentUnitTest {
...
+        allIncidents = incidentService.fetchIncidents(2019, 2020, 40, 1)
+    }
+
+    private fun thenTheIncidentCollectionShouldContainIncidentWithCaseId400434() {
```
Comment (feroxi-s) [inline app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt:117]: Function names now in lowerCamelCase: https://kotlinlang.org/docs/coding-conventions.html#function-names

File: app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt (hunk 4 above)
Comment (discospiff) [reply app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt:102]: Do you have a source for how this reduces technical debt?  We typically prefer a large number of small classes over a small number of large classes... so I would go in the other direction.

**Comment by discospiff** [comment]
From this review, I would harvest the change to lowerCamelCase for test function names.

**Pull request closed without merging**